packages/devtools-reps/src/reps/symbol.js

```diff
@@ -21,12 +24,28 @@ function SymbolRep(props) {
   const { className = "objectBox objectBox-symbol", object } = props;
   const { name } = object;
 
+  let symbolText;
+  if (!name) {
+    symbolText = "";
+  } else if (name.type && name.type === "longString") {
+    symbolText = StringRep({
+      object: name || "",
+      shouldCrop: true,
+      cropLimit: MAX_STRING_LENGTH
+    });
+  } else {
+    const { Rep } = require("./rep");
+    symbolText = Rep({ object: name });
```

Comment:
here `name` should be a string, so we can directly assign it to `symbolText`.

Reply:
Are you suggesting coercing `name` to a string (i.e. `String(name)` or `name+""` or using the String rep?